dotnetv3/Keyspaces/Scenarios/UIMethods.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+namespace KeyspacesBasics;
+
+public class UiMethods
+{
+    public readonly string SepBar = new string('-', Console.WindowWidth);
+
+    /// <summary>
+    /// Show information about the scenario.
+    /// </summary>
+    public void DisplayOverview()
+    {
+        Console.Clear();
+        DisplayTitle("Welcome to the Amazon Keyspaces (for Apache Cassandra) Demo");
+
+        Console.WriteLine("This example application will do the following:");
+        Console.WriteLine("\t 1. Create a keyspace.");
+        Console.WriteLine("\t 2. List existing keyspaces.");
+        Console.WriteLine("\t 3. Create a table to store movie data.");
+        Console.WriteLine("\t 4. Display the table's schema.");
+        Console.WriteLine("\t 5. List all tables in the Amazon keyspace.");
+        Console.WriteLine("\t 6. Pause to allow the user to perform CRUD operations.");
```

Comment:
Is this list still accurate? It seems to do the CRUD operations itself now.

Reply:
Rewrote the description.